### DISCUSSION TITLE
.NET Core `latest` Docker tags updated to 3.1

### DISCUSSION THREAD

**Discussion by mthalman** [dotnet/dotnet-docker#3662] [Announcements]
# .NET Core `latest` Docker tags updated to 3.1

The `latest` tags for the [dotnet/core](https://hub.docker.com/_/microsoft-dotnet-core) repos have been updated to reference [.NET Core 3.1](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-1/).

## Details

* [dotnet-docker#1509](https://github.com/dotnet/dotnet-docker/pull/1509)

The `latest` tags for the [dotnet/core](https://hub.docker.com/_/microsoft-dotnet-core) repos are always intended to reference the latest supported version of .NET Core. At this time, the latest supported version is .NET Core 3.1. This version of .NET Core is also a [long-term supported (LTS) release](https://dotnet.microsoft.com/platform/support/policy/dotnet-core).

The following Docker repos have been updated:

* [dotnet/core/sdk](https://hub.docker.com/_/microsoft-dotnet-core-sdk)
* [dotnet/core/aspnet](https://hub.docker.com/_/microsoft-dotnet-core-aspnet)
* [dotnet/core/runtime](https://hub.docker.com/_/microsoft-dotnet-core-runtime)
* [dotnet/core/runtime-deps](https://hub.docker.com/_/microsoft-dotnet-core-runtime-deps)

It is recommended to use a more-specific tag than `latest` for anything other than casual development. Once you have an application that you believe will eventually go to production, you are recommended to use a tag like `3.1`.

**Comment by 4deeptech** [comment]
I tried yesterday and again today to pull and use the Debian 10 '3.1.100' SDK image and get i/o timeout errors pulling image configuration.  The 3.0.100 image reference does not do this.

Error:
error pulling image configuration: Get https://mcreus0.blob.core.windows.net/aba285c624a04409823b708c7a50e7b9-jttfjm99vo//docker/registry/v2/blobs/sha256/98/9817c25953a8ede59da1b397aa096c5ba269bbd969f55f1819393dda35f9d19f/data?se=2019-12-06T22%3A24%3A25Z&sig=n9whbNNyggFZrjyz9yyFU60hF3bgyf4yYno%2FVqpsZzM%3D&sp=r&sr=b&sv=2016-05-31&regid=aba285c624a04409823b708c7a50e7b9: dial tcp: lookup mcreus0.blob.core.windows.net on 192.168.65.1:53: read udp 192.168.65.3:43777->192.168.65.1:53: i/o timeout

Image reference being used is:

```FROM mcr.microsoft.com/dotnet/core/sdk:3.1.100```

Anyone else run into this?

**Comment by MichaelSimons** [comment]
@4deeptech , I have not had any issues.  I'd suggest reporting the issue at https://github.com/microsoft/containerregistry.

**Comment by 4deeptech** [comment]
This is using VSCode remote containers with Docker Desktop. I just now switched my docker file back to:

```FROM mcr.microsoft.com/dotnet/core/sdk:3.0.100```

and it fired right up.

I will report this over there and see if anyone knows anything about this.  Thanks.
